pkg/utils/utils.go

```diff
@@ -25,16 +25,16 @@ import (
 	"golang.org/x/exp/rand"
 )
 
-func RandDate(rnd *rand.Rand) string {
-	return RandTime(rnd).Format("2006-01-02")
+func RandDate(rnd *rand.Rand) time.Time {
+	min := time.Date(1970, 1, 1, 0, 0, 0, 0, time.UTC).UnixMilli()
+	max := time.Date(2024, 1, 1, 0, 0, 0, 0, time.UTC).UnixMilli()
```

Comment:
Why exactly this date is max date?

Reply:
It`s my predisposition to round numbers and ranges:) It may be changed